docs/build-apps/apps.md

```diff
@@ -19,7 +19,7 @@ To build a payment-based application, a firm understanding of how to construct a
 # Tokenizing Assets or Creating Your Own Coin
 Asset-based applications can support a diverse set of use cases, but by definition, they all require a standardized, digital representation of a real-world asset on the blockchain that is either [fungible](https://en.wikipedia.org/wiki/Fungibility) (one of many) or [non-fungible](https://en.wikipedia.org/wiki/Non-fungible_token) (unique). Assets can represent any asset like company shares, in-game cosmetics, loyalty points, ownership in real estate, and stable coins (like USDT). They can also represent more abstract concepts like the rights to a song or a type of certification or permission giving the holding account access to certain applications or parts of applications. This list is hardly exhaustive, but is meant to show the range of possibilities and use cases that asset-based applications can support.
 
-Algorand Standard Assets are the underlying technology that support these applications on Algorand and are deployed at Layer-1 through a simple transaction (no contract code is required). Assets can be traded like the Algo -  Algorand’s native token. They are also highly configurable, providing capabilities for minting, freezing, and revoking tokens.  For more information on building and deploying asset-based applications see the [Algorand Standard Assets](../features/asa.md) feature guide.
```

Comment:
this fix is incorrect - a dash is needed here

Reply:
I replaced the hyphen ("-") with an em-dash ("—"), though it's hard to tell in this monospaced font. That is, I changed "the Algo-Algorand's native token" to "the Algo—Algorand's native token". The former looks like a hyphenated noun (an "Algo-Algorand") whereas the latter is clearly a break in the sentence.

Reply:
ok, but maybe keep spaces around?

Reply:
Okay, done. (Though I prefer it without the spaces. 🙃)